CHANGES.rst

```diff
@@ -18,6 +18,110 @@
 Changes
 =======
 
+` <https://github.com/SwissDataScienceCenter/renku-python/compare/v0.11.6...v0.11.7>`__ (2020-11-03)
```

Comment:
We probably should mark this as `0.12.0` since we have a bit more than bug fixes 😉.